### PR TITLE
Update status and remove version

### PR DESCRIPTION
**Pull request by Echolon**
@wurstsalat3000

The only thing I don't understand yet:

- All namespaces should go in rdf:RDF

Ref: https://github.com/monal-im/Monal/pull/501#issuecomment-774440363

### CODE REVIEW

**Comment by wurstsalat3000** [comment]
I *think* this is how it should look like:
```
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
        xmlns:xmpp="https://linkmauve.fr/ns/xmpp-doap#"
        xmlns="http://usefulinc.com/ns/doap#"
        xmlns:schema="https://schema.org/" 
  <Project>
```
Additional note: git repository location also points to the old URL.

**Comment by Echolon** [comment]
Thank you @wurstsalat3000

**Comment by tmolitor-stud-tu** [comment]
@Echolon can you extract a list of XEPs that are marked "planned", I want to check if this lists something we are not planning to do...

**Comment by Echolon** [comment]
> @Echolon can you extract a list of XEPs that are marked "planned", I want to check if this lists something we are not planning to do...

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0027: Current Jabber OpenPGP Usage</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0033: Extended Stanza Addressing</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0047: In-Band Bytestreams</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0048: Bookmarks</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0054: vcard-temp</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0055: Jabber Search</xmpp:note>  

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0065: SOCKS5 Bytestreams</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0070: Verifying HTTP Requests via XMPP</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0107: User Mood</xmpp:note> 

        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0153.html" />
        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0153: vCard-Based Avatars</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0215: External Service Discovery</xmpp:note>  

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0234: Jingle File Transfer</xmpp:note> 

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0249: Direct MUC Invitations</xmpp:note> 

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0261: Jingle In-Band Bytestreams Transport 

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0308: Last Message Correction</xmpp:note> 

        <xmpp:status>complete</xmpp:status>
        <xmpp:note>XEP-0357: Push Notifications</xmpp:note> 

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0369: Mediated Information eXchange (MIX)</xmpp:note> 

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0374: OpenPGP for XMPP Instant Messaging</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0377: Spam Reporting (via XEP-0191)</xmpp:note>

        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0379.html" />
        <xmpp:status>planned</xmpp:status>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0380: Explicit Message Encryption</xmpp:note>   

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0386: Bind 2.0</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0390: Entity Capabilities 2.0</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0392: Consistent Color Generation</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0396: Jingle Encrypted Transports - OMEMO</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0397: Instant Stream Resumption</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0402: PEP Native Bookmarks</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0409: IM Routing-NG</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0410: MUC Self-Ping (Schrödinger's Chat)</xmpp:note>

        <xmpp:status>planned</xmpp:status>
        <xmpp:note>XEP-0420: Stanza Content Encryption</xmpp:note>

**Comment by tmolitor-stud-tu** [comment]
That are many "planned" that are not really planned

**Comment by tmolitor-stud-tu** [comment]
planned is only:
```

<xmpp:status>planned</xmpp:status>
<xmpp:note>XEP-0048: Bookmarks</xmpp:note>

<xmpp:status>planned</xmpp:status>
<xmpp:note>XEP-0215: External Service Discovery</xmpp:note>  

<xmpp:status>planned</xmpp:status>
<xmpp:note>XEP-0249: Direct MUC Invitations</xmpp:note>

<xmpp:status>planned</xmpp:status>
<xmpp:note>XEP-0402: PEP Native Bookmarks</xmpp:note>
```

this is already implemented:

```
<xmpp:status>complete</xmpp:status>
<xmpp:note>XEP-0308: Last Message Correction</xmpp:note>
```

**Comment by Echolon** [comment]
Ready for review

File: monal.doap [hunk 1]
```diff
@@ -41,488 +41,427 @@
...
+        <xmpp:status>partial</xmpp:status>
         <xmpp:note>XEP-0066: Out of Band Data</xmpp:note>
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0070.html" />
-        <xmpp:status>no</xmpp:status>
-        <xmpp:version>?</xmpp:version>
+        <xmpp:status>planned</xmpp:status>
```
Comment (tmolitor-stud-tu) [inline monal.doap:135]: not planned and never was

File: monal.doap [hunk 2]
```diff
@@ -41,488 +41,427 @@
...
-        <xmpp:version>?</xmpp:version>
         <xmpp:note>XEP-0184: Message Receipts</xmpp:note>     
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0191.html" />
-        <xmpp:status>partly</xmpp:status>
-        <xmpp:version>?</xmpp:version>
+        <xmpp:status>partial</xmpp:status>
```
Comment (tmolitor-stud-tu) [inline monal.doap:226]: now complete in version 5.0

File: monal.doap [hunk 3]
```diff
@@ -41,488 +41,427 @@
...
+        <xmpp:status>wontfix</xmpp:status>
         <xmpp:note>XEP-0027: Current Jabber OpenPGP Usage</xmpp:note>
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0030.html" />
-        <xmpp:status>complete</xmpp:status>
-        <xmpp:version>?</xmpp:version>
+        <xmpp:status>wontfix</xmpp:status>
```
Comment (tmolitor-stud-tu) [inline monal.doap:58]: sorry, this is implemented

File: monal.doap [hunk 4]
```diff
@@ -41,488 +41,427 @@
...
-        <xmpp:version>?</xmpp:version>
         <xmpp:note>XEP-0368: SRV records for XMPP over TLS</xmpp:note> 
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0369.html" />
-        <xmpp:status>no</xmpp:status>
-        <xmpp:version>?</xmpp:version>
+        <xmpp:status>wontfix</xmpp:status>
```
Comment (tmolitor-stud-tu) [inline monal.doap:352]: could be planned, too...but far into the future

File: monal.doap [hunk 5]
```diff
@@ -41,488 +41,427 @@
...
         <xmpp:note>XEP-0369: Mediated Information eXchange (MIX)</xmpp:note> 
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0374.html" />
-        <xmpp:status>no</xmpp:status>
-        <xmpp:version>?</xmpp:version
+        <xmpp:status>wontfix</xmpp:status>
```
Comment (tmolitor-stud-tu) [inline monal.doap:359]: could be planned, too...but far into the future

File: monal.doap [hunk 6]
```diff
@@ -41,488 +41,427 @@
...
         <xmpp:note>XEP-0377: Spam Reporting (via XEP-0191)</xmpp:note>
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0379.html" />
-        <xmpp:status>no</xmpp:status>
-        <xmpp:version>?</xmpp:version>
+        <xmpp:status>wontfix</xmpp:status>
         <xmpp:note>XEP-0379: Pre-Authenticated Roster Subscription</xmpp:note>
```
Comment (tmolitor-stud-tu) [inline monal.doap:374]: partial

File: monal.doap [hunk 7]
```diff
@@ -41,488 +41,427 @@
...
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0380.html" />
-        <xmpp:status>no</xmpp:status>
-        <xmpp:version>?</xmpp:version>
+        <xmpp:status>wontfix</xmpp:status>
```
Comment (tmolitor-stud-tu) [inline monal.doap:380]: planned

File: monal.doap [hunk 8]
```diff
@@ -41,488 +41,427 @@
...
         <xmpp:note>XEP-0390: Entity Capabilities 2.0</xmpp:note>
       </xmpp:SupportedXep>
     </implements>
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0392.html" />
-        <xmpp:status>no</xmpp:status>
-        <xmpp:version>?</xmpp:version>
+        <xmpp:status>wontfix</xmpp:status>
         <xmpp:note>XEP-0392: Consistent Color Generation</xmpp:note>
```
Comment (tmolitor-stud-tu) [inline monal.doap:409]: planned

**Review by tmolitor-stud-tu — CHANGES_REQUESTED** [review]

**Comment by Echolon** [comment]
Added changes

File: monal.doap [hunk 9]
```diff
@@ -223,7 +223,7 @@
     <implements>
       <xmpp:SupportedXep>
         <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0191.html" />
-        <xmpp:status>partial</xmpp:status>
+        <xmpp:status>complete</xmpp:status>
         <xmpp:note>XEP-0191: Blocking Command. Partly, missing sync with server which accounts are blocked.</xmpp:note>
```
Comment (tmolitor-stud-tu) [inline monal.doap:227]: please remove the partly... part in the description

**Review by tmolitor-stud-tu — CHANGES_REQUESTED** [review]

File: monal.doap (hunk 9 above)
Comment (Echolon) [reply monal.doap]: Done

**Review by tmolitor-stud-tu — APPROVED** [review]

**Pull request merged**